README.md

```diff
@@ -69,7 +69,7 @@ Once you have your Swift package set up, adding **MeiliSearch-Swift** as a depen
 
 ```swift
 dependencies: [
-    .package(url: "https://github.com/meilisearch/meilisearch-swift.git")
+    .package(url: "https://github.com/meilisearch/meilisearch-swift.git", from: "0.1.0")
```

Comment:
Does this represent the version of meilisearch-swift?

Reply:
Yes. Specifying a package without from: (or some other constraint) only works with local paths ([see docs](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#package-dependency)).

Since the version is specified as from, up to next major version it means the current setting will satisfy all versions until 1.x is released. So it follows that this should be updated with each major version release.